### ISSUE TITLE
MIME type is 'application/gzip' instead of 'application/javascript'

### ISSUE THREAD

**Issue by BeaRRRRR** [tkoenig89/express-static-gzip#35]
I want by backend express server to serve gziped bundle.js.

My server.js (I included only the part responsible for serving gzip)
```
app.use("/", expressStaticGzip(DIST_DIR, {}));
```

My index.html
```
<html>
  <head>
    <meta charset="UTF-8" />
    <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width" />
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
    <title>Architects of Belarus</title>
  </head>
  <body>
    <div id="app"></div>
    <script type="text/javascript" src="bundle.js.gz"></script></body>
</html>
```

When I load the page, there are the following errors in the console:
```
The script from “http://localhost:3000/bundle.js.gz” was loaded even though its MIME type (“application/gzip”) is not a valid JavaScript MIME type.
localhost:3000
SyntaxError: illegal character
bundle.js.gz:1
```
![2020-02-27-205549_1920x1080_scrot](https://user-images.githubusercontent.com/32522878/75471891-d5384f00-59a3-11ea-95b5-e4718a08658e.png)

I believe that this problem would be solved if I could set the MIME type to 'application/javascript'

**Comment by tkoenig89** [comment]
I did some testing and everything seems to be correct with `express-static-gzip`'s behaviour. Files get served with the correct content-type, even so the gziped variant is served from the backend.

But I think, I spoted the culprit. Your `index.html` should request the `bundle.js` file, not the `bundle.js.gz` file. Serving the gziped version, if the browser can handle it, is completly handled by the backend, and should not be hardcoded into your html code.

So to get rid of the warning I suggest you replace
```
<script type="text/javascript" src="bundle.js.gz"></script>
``` 
with 
```
<script type="text/javascript" src="bundle.js"></script>
```
